src/nodes/utils.ts

```diff
@@ -1,29 +1,75 @@
-import type { NodeData, NodeId, NodeIdEncoded } from './types';
+import type {
+  NodeId,
+  NodeIdEncoded,
+  NodeBucket,
+  NodeBucketIndex,
+} from './types';
 import { IdInternal } from '@matrixai/id';
-import { bytes2BigInt } from '../utils';
+import lexi from 'lexicographic-integer';
+import { bytes2BigInt, bufferSplit } from '../utils';
+import * as keysUtils from '../keys/utils';
+
+// FIXME:
```

Comment:
Shoudl look for all the FIXMEs and resolve them. Also remove any spurious commentary.